### ISSUE TITLE
Page Not Loading

### ISSUE THREAD

**Issue by RIB0FLAVIN** [konrul/Pi-Kitchen-Dashboard#1]
I can't for the life of my figure out what I'm doing wrong.

I've put in my Yahoo Consumer Key, Consumer Secret, and App ID, and new WOEID, but it still doesn't load.  What else would I be missing?

Any help/suggestions would be greatly appreciated.

**Comment by imchesterhk** [comment]
i got the same issue ... is it something wrong with the Callback Domain when i am applying the API at yahoo ? i didnt input anything

**Comment by imchesterhk** [comment]
but when i tried to node the fetch.js , there are some latest info shown up, just not show up at the browser

**Comment by konrul** [comment]
Check:
* Yahoo have approved api usage for your account
* Check that `/var/www/html/Pi-Kitchen-Dashboard/data/weather.json` is being written and the content looks sensible, and the file has been written to recently
* Check you can open the file in your browser http://something/Pi-Kitchen-Dashboard/data/weather.json
* Check the Web Developer Web Console log in your browser

**Comment by imchesterhk** [comment]
> Check:
> 
> * Yahoo have approved api usage for your account
> * that `/var/www/html/Pi-Kitchen-Dashboard/data/weather.json` is being written and the content looks sensible, and the file has been written to recently
> * you can open the file in your browser http://something/Pi-Kitchen-Dashboard/data/weather.json
> * the Web Developer Web Console log in your browser

Thank you so much for your reply !

- i think yahoo did approved the api, when i tried to ssh
 node /var/www/html/Pi-Kitchen-Dashboard/js/fetch.js
there is some latest info shown up by compare , i guess its already done with that ?

-i am not sure what that is, because i dont have the data folder under my Pi-Kitchen-Dashboard folder, do you mean "Pi-Kitchen-Dashboard/js/weather.js" ? and if yes, i can access it on my browser and no error. (just show the content of that js file.)

- when i was using chrome to open it, it does have some error, i should check that before.

weather.js:112 Uncaught ReferenceError: $ is not defined
    at weather.js:112
    at weather.js:194
(anonymous) @ weather.js:112
(anonymous) @ weather.js:194

also when i am using ssh to node the weather.js
it shown
-----------------
root@raspberrypi:~# node /var/www/html/Pi-Kitchen-Dashboard/js/weather.js
/var/www/html/Pi-Kitchen-Dashboard/js/weather.js:112
                $(document).ready(function() {
                ^

ReferenceError: $ is not defined
    at /var/www/html/Pi-Kitchen-Dashboard/js/weather.js:112:3
    at Object.<anonymous> (/var/www/html/Pi-Kitchen-Dashboard/js/weather.js:194:2)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:266:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:596:3)
-----------------
i didnt touch those coding, is there something you may think of i can fix it ? 

No matter what, thanks again for your help !

**Comment by konrul** [comment]
Your cron job calls `node /var/www/html/Pi-Kitchen-Dashboard/js/fetch.js` and writes it to `/var/www/html/Pi-Kitchen-Dashboard/data/weather.json`

`weather.js` is not involved.

Please run through the troubleshooting steps above. Also check you have not overwritten `weather.js`

**Comment by imchesterhk** [comment]
> Your cron job calls `node /var/www/html/Pi-Kitchen-Dashboard/js/fetch.js` and writes it to `/var/www/html/Pi-Kitchen-Dashboard/data/weather.json`
> 
> `weather.js` is not involved.
> 
> Please run through the troubleshooting steps above. Also check you have not overwritten `weather.js`

Thanks again for the help !
now i understand, and i have created a data folder manually, after that the cron.sh successfully create a weather.json under data folder, and the data inside are downloaded correctly from yahoo !

but so weird... the same error still shown up ... btw one thing i didnt mention before, is i also dont have the clock showing, do you think is it somthing wrong with my nginx or other system problem ?

here is the F12 on chrome
https://imgur.com/a/J6XNJSF

**Comment by imchesterhk** [comment]
<script src="../../bower_components/jquery/dist/jquery.min.js"></script>
			<script src="../../bower_components/moment/min/moment.min.js"></script>

also in index.html,  these two line have error
there is no such bower_components folder. i guess that is something wrong with my system and not related to this program ? Thanks again !

**Comment by imchesterhk** [comment]
YES ! after i do the bower install again , all working now ! Thanks again !

**Comment by konrul** [comment]
@RIB0FLAVIN is it working for you?

**Comment by RIB0FLAVIN** [comment]
> 
> 
> @RIB0FLAVIN is it working for you?

I had forgotten to fill out the form for Yahoo to approve api usage.  I did that yesterday, and waiting for the approval.  I'm hoping that is my only issue.

I'll report back if it's not, and also try the other troubleshooting steps mentioned above.

I really appreciate the help.
